apps/main/src/llamalend/features/market-list/chips/MarketFilterChipWrapper.tsx

```diff
@@ -16,13 +20,25 @@ type MarketsFilterChipsProps = {
   resetFilters: () => void
   hasFilters: boolean
   children?: ReactNode
-}
+  userHasPositions: LlamaMarketsResult['userHasPositions'] | undefined
+  hasFavorites: boolean | undefined
+  onSortingChange?: OnChangeFn<SortingState>
+  sortField?: LlamaMarketColumnId
+  data: LlamaMarket[]
+  minLiquidity?: number
```

Comment:
There is only 1 usage of this component, preferably let's keep them required until we have a use case
```suggestion
  onSortingChange: OnChangeFn<SortingState>
  sortField: LlamaMarketColumnId
  data: LlamaMarket[]
  minLiquidity: number
```